tests/bsim/bluetooth/audio_samples/broadcast_audio_sink/Kconfig.sysbuild

```diff
@@ -8,3 +8,4 @@ config NATIVE_SIMULATOR_PRIMARY_MCU_INDEX
 	# Let's pass the test arguments to the application MCU test
 	# otherwise by default they would have gone to the net core.
 	default 0 if $(BOARD) = "nrf5340bsim_nrf5340_cpuapp"
```

Comment:
this line can be updated instead

Reply:
Correct. Though these will be removed in a follow up commit. So I hope you don't mind that provisionally we hav two lines instead of an OR

Reply:
Sure